tests/pages/firefox/enterprise.py

```diff
@@ -11,8 +11,18 @@ class EnterprisePage(FirefoxBasePage):
 
     URL_TEMPLATE = '/{locale}/firefox/enterprise/'
 
-    _primary_cta_locator = (By.CSS_SELECTOR, '.qa-cta')
+    _signup_button_locator = (By.CSS_SELECTOR, '.qa-sign-up')
```

Comment:
Oops, I missed this... the old button is still present with the old class so this test is breaking https://github.com/mozilla/bedrock/pull/5695/files#diff-a00ca8c8275931bd72c6b61eb83ab201R34

Fix incoming.